edx_proctoring/tests/test_views.py

```diff
@@ -1877,6 +1877,50 @@ def test_review_callback_get(self):
 
         self.assertEqual(response.status_code, 405)
 
+    def test_attempt_review_status_callback(self):
+        """
+        Test the ProctoredExamAttemptReviewStatus view
+        """
+        proctored_exam = ProctoredExam.objects.create(
+            course_id='a/b/c',
+            content_id='test_content',
+            exam_name='Test Exam',
+            external_id='123aXqe3',
+            time_limit_mins=30,
+            is_proctored=True,
+            due_date=datetime.now(pytz.UTC) + timedelta(minutes=40)
+        )
+
+        attempt = ProctoredExamStudentAttempt.objects.create(
+            proctored_exam=proctored_exam,
+            user=self.user,
+            allowed_time_limit_mins=30,
+            taking_as_proctored=True,
+            external_id=proctored_exam.external_id,
+            status=ProctoredExamStudentAttemptStatus.started
+        )
+
+        response = self.client.put(
+            reverse(
+                'edx_proctoring.proctored_exam.attempt.review_status',
+                args=[attempt.id]
+            ),
+            {},
+            content_type='application/json'
+        )
+
+        self.assertEqual(response.status_code, 200)
+
+        response = self.client.put(
+            reverse(
+                'edx_proctoring.proctored_exam.attempt.review_status',
+                args='5'
+            ),
+            {},
+            content_type='application/json'
+        )
+        self.assertEqual(response.status_code, 400)
```

Comment:
I think it's better to move the 400 case into a separate test case where you can actually validates the exception that has been raised when hitting the url with the invalid attempt_id
`with self.assertRaises(StudentExamAttemptDoesNotExistsException)`